Exchange-Guide.md

```diff
@@ -31,6 +31,15 @@ Default is `matching` mode, which means that `xud` acts as order book & matching
 * `SubscribeSwaps` informs about successful/failed swaps with other peers in an event stream and is to be used with `PlaceOrder`. Use this call to get real-time swap events from placed orders to track order executions, update your user's balances, and inform users when one of their orders was settled.
 * `ExecuteSwap` executes a swap for an match your internal matching engine found with a remote peer. This call requires `orderID`, `pairID` & `quantity`. You can continue using your internal `orderID`s, `xud` takes care of mapping such with `orderID`s in the Exchange Union network. This is a sync call, returning the final result of a swap. It might take several seconds for the swap to complete.
 
+### Hints
+* `xud` doesn't *forward* orders, it only sends to and receives from direct peers. Reason for this, among others, are manipulation and speed. A more scalable approach will be [introduced in a later stage](https://github.com/ExchangeUnion/Docs/blob/master/XU-TechnicalPaper.md#xus-dob-protocol-has-the-following-goals).
+* Restarting or stopping `xud` clears all orders. When `xud` boots up, it automatically reconnects to its peers and fetches all open orders.
+* `xud` relies on is it's own (local) view of orders. This is also how it determines if it is a maker or a taker when a match is found. If the peer order reached the order book first = maker, if the local order reached the order book first = taker. Due to network latency it now can happen that both parties determine to be maker (or taker). In this case, a swap won't be initiated.
```

Comment:
> If the peer order reached the order book first = maker

Yes, if you're referring to the peer order, and not to `xud` (the taker), which was mentioned in the previous sentence. If so, perhaps this should be changed to something like "If the peer order reached the order book first, _then it's the maker order_".

> if the local order reached the order book first = taker.

A local order doesn't really reach the local order book (for swap purposes). It reaches remote order books, as a maker.

The differentiation here between own/peer order is problematic, because simply whatever reaches the order book first, is the maker. 

It worth to mention that only the taker side initiate the swap, and that's why if both parties determine to be the maker, the swap won't get initiated. 

In addition, for the same reason, peer orders aren't getting matched upon arrival to the local order book, because if they would then they would be a taker order, which the local `xud` don't initiate swaps for. 
That's why they are used only as maker orders, and local orders are used only as taker orders.

Reply:
> Yes, if you're referring to the peer order, and not to `xud` (the taker), which was mentioned in the previous sentence. If so, perhaps this should be changed to something like "If the peer order reached the order book first, _then it's the maker order_".

No, I wanted to refer to `xud`, but screwed it up. Fixed.

> A local order doesn't really reach the local order book (for swap purposes). It reaches remote order books, as a maker.

Yeah but I don't think it's necessary to say that. Added that the taker initiates the swap tho, that should clarify that maker orders just sit there and wait for swaps being initiated by a remote taker.